src/components/ConfirmationModal/ConfirmationModal.jsx

```diff
@@ -0,0 +1,19 @@
+import "./ConfirmationModal.css";
+
+function ConfirmationModal({ isOpen, handleCloseClick, text }) {
+  return (
+    <div className={`modal modal_type_${name} ${isOpen ? "modal_opened" : ""}`}>
+      <div className="modal__container">
+        <button
+          className="modal__close-button"
+          type="button"
+          aria-label="close"
+          onClick={handleCloseClick}
+        />
+        <h2 className="confirmation-modal__heading">{text}</h2>
```

Comment:
`confirmation-modal__heading`


There is a BEM mistake

You don't have `confirmation-modal` block in this component. You can use `modal__heading` here or `modal__confirmation-heading`

More info about `BEM` naming can be found here: https://en.bem.info/methodology/quick-start/